week-4/Homework/mandatory/3-water-bottle.js

```diff
@@ -12,12 +12,15 @@ let bottle = {
   volume: 0,
   fill: function() {
     // calling this function should make you bottles volume = 100;
+    return (this.volume = 100);
   },
   drink: function() {
     // calling this function should decrease your bottles volume by 10;
+    return (this.volume -= 10);
   },
   empty: function() {
     // this function should return true if your bottles volume = 0
+    if (this.volume == 0) return true;

```

Comment:
This will work for some of the cases but all of them. What if the volume is not 0?